apps/web/src/components/AddressMevLookup.tsx

```diff
@@ -11,6 +11,7 @@ import { socials, SocialType } from "@/constants/links";
 import { signInWithTwitter } from "@/services/supabase";
 import useSupabaseSession from "@/hooks/useSupabaseSession";
 import { LocalStorage } from "@/constants/storage";
+import clsx from "clsx";
```

Comment:
If we ever run into issues where tailwind styles are clashing and overwriting each other then https://github.com/dcastil/tailwind-merge is super useful for that